ui-tests/cypress/support/pageObjects/pages/designer/widgetconfigs/ContentListWidgetConfigPage.js

```diff
@@ -10,6 +10,10 @@ export default class ContentListWidgetConfigPage extends WidgetConfigPage {
     return this.getContentListTableBody().find('td').contains(title).siblings();
   }
 
+  getAddButtonFromTableRowWithTitle(title) {
+    return this.getContentListTableBody().find('td').contains(title).siblings().find('button').contains(/^Add$/);
```

Comment:
Please, do not use `contains`

Reply:
hi @g-di-martino , I have added more specificity to the statement here. this is to select a particular `<button>` within the row selected. I have also used the title of the row entry to select a row. If you can consider using the contains command, that would be great

Reply:
It's not about specificity, it's for reliability: texts can change without affecting the functionality, which is what we are testing, making the test fail for no reason.
`contains` is implicitly asserting that the text is correct which is not the scope of our tests and, above all, is not a structural information.
If you cannot find any other mean to identify the element leave it, we'll improve it later, but at least put a comment on it (TODO or FIXME) to remind us it's a weak spot.

Reply:
best thing we can add support these are amending test id attributes (aka `data-testid`) for the said elements. yeah, I will add the comments in the next commit.